src/model/enums/rundown-event-type.ts

```diff
@@ -1,10 +1,10 @@
 export enum RundownEventType {
-  ACTIVATE = 'ACTIVATE',
-  DEACTIVATE = 'DEACTIVATE',
+  ACTIVATED = 'ACTIVATED',
+  DEACTIVATED = 'DEACTIVATE',
+  DELETED = 'DELETED',
   RESET = 'RESET',
-  TAKE = 'TAKE',
+  TAKEN = 'TAKE',
```

Comment:
Please keep the name of the enum attribute consistent with the value of the enum attribute.
If you change one, then also change the other.

Reply:
Is changed.